src/docx/assets/zarzadzenie/word/document.xml

```diff
@@ -0,0 +1,150 @@
+  xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" 
+  xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math" 
+  xmlns:v="urn:schemas-microsoft-com:vml" 
+  xmlns:wp="http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing" 
+  xmlns:w10="urn:schemas-microsoft-com:office:word" 
+  xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main" 
+  xmlns:wne="http://schemas.microsoft.com/office/word/2006/wordml"
+  xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture">
+  <w:body>
+    <w:p>
+      <w:pPr>
+        <w:pStyle w:val="centerAlignedTitleGreenColor"/>
+      </w:pPr>
+      <w:r>
+        <w:t>Poradnia Psychologiczno – Pedagogiczna we Wrześni</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="100"/>
+        <w:pStyle w:val="Uppercase"/>
+      </w:pPr>
+      <w:r>
+        <w:t>PPP</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:pStyle w:val="City-and-date"/>
+      </w:pPr>
+      <w:r>
+        <w:t>{{ city }}, {{ date }}</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="500"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>Zarządzenie nr {{ no }}/2020</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>dyrektora Poradni z dnia {{ date }} r.</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>w sprawie powołania zespołu orzekającego</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>§1</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="normalParagraph"/>
+      </w:pPr>
+      <w:r>
+        <w:t xml:space="preserve">Dyrektor Poradni Psychologicznej w {{ supportCenter.city }}, {{ supportCenter.post }}, {{ supportCenter.address }}, na podstawie § 4 ust. 1-6 rozporządzenia Ministra Edukacji Narodowej z dnia 7 września 2017 r. w sprawie orzeczeń i opinii wydawanych przez zespoły orzekające działające w publicznych poradniach psychologiczno – pedagogicznych, (Dz. U. z 2017 r. poz. 1743) powołuje zespół orzekający do rozpatrzenia wniosku </w:t>
+      </w:r>
+      <w:r>
+        <w:rPr>
+          <w:b/>
+        </w:rPr>
+        <w:t xml:space="preserve">{{ applicant.name }} </w:t>
+      </w:r>
+      <w:r>
+        <w:t xml:space="preserve">zam. </w:t>
+      </w:r>
+      <w:r>
+        <w:rPr>
+          <w:b/>
+        </w:rPr>
+        <w:t xml:space="preserve">{{ applicant.street }}/{{ applicant.houseNumber }}, {{ applicant.postalCode }} {{ applicant.city }} </w:t>
```

Comment:
Why `/`?

Reply:
`street` property in applicant object contains 2 information - the name of the street and the street number. `/` is a separator between street number and house number. <br>
I would leave this char as it is right now.

Reply:
We have a mess with that address data... I think that this is not a big deal now. But more important is to solve that issue (I mentioned it on the very begin of my review), with `style.xml` file, because with our `generateDocument` function `common` style xml would be used - not that one you commit.

Reply:
I've refactored `style.xml` and template files. Redundant styles have been removed from file. Thus, template uses styles defined previously. In one case I had to prepare new style configuration for this template.